assets/js/game.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-var */
+const GameBoard = (() => {
+  let boardArray = ['', '', '', '', '', '', '', '', ''];
+
+  const getBoard = function getBoard() {
+    return boardArray;
+  };
+
+  const updateBoard = function updateBoard(index, sign) {
+    boardArray[index] = sign;
+  };
+
+  const clearBoard = function clearBoard() {
+    boardArray = ['', '', '', '', '', '', '', '', ''];
+  };
+
+  return {
+    get: getBoard,
+    update: updateBoard,
+    clear: clearBoard,
+  };
+})();
+
+
+const GamePlay = (() => {
+  var gameStatus = false;
+  var playerOne;
+  var playerTwo;
+  var playerOneScore = 0;
+  var playerTwoScore = 0;
+  var turn = 0;
+  let playerOneWin = false;
+  let playerTwoWin = false;
+
+  const getPlayers = function getPlayers() {
+    const name1 = document.getElementById('player1').value || 'Player1';
+    const name2 = document.getElementById('player2').value || 'Player2';
```

Comment:
**`OPTIONAL`** Move those two lines into `DOM.js` file just for continuing separating.